pypeit/core/flexure.py

```diff
@@ -313,7 +313,7 @@ def spec_flex_shift(obj_skyspec, arx_skyspec, arx_fwhm_pix, spec_fwhm_pix=None,
 
     return dict(polyfit=fit, shift=shift, subpix=subpix_grid,
                 corr=corr[subpix_grid.astype(int)], sky_spec=obj_skyspec, arx_spec=arx_skyspec,
-                corr_cen=corr.size/2, smooth=smooth_fwhm_pix, method=method)
+                corr_cen=lag0, smooth=smooth_fwhm_pix, method=method)
```

Comment:
@kbwestfall said: It would be good to have someone else more familiar with this part of the code to look at this. @profxj maybe? Note that `lag0 = corr.size//2` is slightly different from `corr.size/2`, but I don't know if this makes a difference.

Reply:
Yes, these are different (by 0.5 pix), and we want `//2`.  The only place where `corr_cen` is used is in the QA plots.  I tracked this down when looking at the parabolas in the `*_spec_flex_corr.png` plots and noticed that the peak of the parabola was offset from the computed flexure shift by about 0.5 pix.  Digging through the code, the data points and parabola in the plot are based off `corr_cen`, which _should_ be the zero-lag center of the correlation function.  By making this change, the parabolas now line up with the computed flexure shift (which _is_ computed with `lag0` -- line 266).

Reply:
This should be fine too. Besides, `corr_cen` is only used for the QA plot and I don't think it would make a lot of difference between `corr.size//2` and `corr.size/2`

Reply:
It is only a visual offset, but it bugged me.